### ISSUE TITLE
Cannot read property 'original' of undefined

### ISSUE THREAD

**Issue by urbany** [rwjblue/ember-template-lint#170]
Hi, just updated:
```diff
-    "ember-cli-template-lint": "0.5.0",
+    "ember-cli-template-lint": "0.5.2",
```

And started seeing multiple errors like this in different templates, I created a minimal hbs to replicate the error:

Template
```hbs
<div>
{{! some comment here }}
  {{tether-popover}}
</div>
```
Error
```
file changed components/my-component.hbs

undefined: Cannot read property 'original' of undefined (modules/forms/templates/components/my-component): 
`TypeError: Cannot read property 'original' of undefined
    at BasePlugin.BlockIndentation.validateBlockChildren (project/node_modules/ember-template-lint/lib/rules/lint-block-indentation.js:246:31)
    at BasePlugin.BlockIndentation.process (project/node_modules/ember-template-lint/lib/rules/lint-block-indentation.js:118:10)
    at BasePlugin.ElementNode (project/node_modules/ember-template-lint/lib/rules/lint-block-indentation.js:109:14)
    at project/node_modules/ember-template-lint/lib/rules/base.js:290:21
    at visitNode (project/node_modules/glimmer-engine/dist/node_modules/glimmer-syntax/lib/traversal/traverse.js:8:32)
    at visitArray (project/node_modules/glimmer-engine/dist/node_modules/glimmer-syntax/lib/traversal/traverse.js:63:22)
    at visitKey (project/node_modules/glimmer-engine/dist/node_modules/glimmer-syntax/lib/traversal/traverse.js:46:9)
    at visitNode (project/node_modules/glimmer-engine/dist/node_modules/glimmer-syntax/lib/traversal/traverse.js:24:13)
    at Object.traverse (project/node_modules/glimmer-engine/dist/node_modules/glimmer-syntax/lib/traversal/traverse.js:105:5)
    at BasePlugin.transform (project/node_modules/ember-template-lint/lib/rules/base.js:76:17)
    at Object.preprocess (project/node_modules/glimmer-engine/dist/node_modules/glimmer-syntax/lib/parser.js:24:31)
    at precompile (project/node_modules/glimmer-engine/dist/node_modules/glimmer-compiler/lib/compiler.js:37:32)
    at Linter.verify (project/node_modules/ember-template-lint/lib/index.js:94:7)
    at TemplateLinter.processString (project/node_modules/ember-cli-template-lint/broccoli-template-linter.js:91:28)
    at Promise.then.result.output (project/node_modules/broccoli-persistent-filter/lib/strategies/persistent.js:41:23)
    at initializePromise (project/node_modules/rsvp/dist/rsvp.js:588:5)
    at new Promise (project/node_modules/rsvp/dist/rsvp.js:1076:31)
    at project/node_modules/broccoli-persistent-filter/lib/strategies/persistent.js:40:18
    at tryCatch (project/node_modules/rsvp/dist/rsvp.js:538:12)
    at invokeCallback (project/node_modules/rsvp/dist/rsvp.js:553:13)
    at publish (project/node_modules/rsvp/dist/rsvp.js:521:7)
    at flush (project/node_modules/rsvp/dist/rsvp.js:2373:5)`
===== 1 Template Linting Error


Build successful - 4263ms.
```

This template does not print the error:
```hbs
<div>
  {{! some comment here }}
  {{tether-popover}}
</div>
```

**Comment by rwjblue** [comment]
0.5.2 is a bit older and there are many more patch releases after 0.5.2 that might have fixed this bug, would you mind testing updating to the latest and see if that fixes the issue?

If not a failing test case PR would be super helpful...

**Comment by urbany** [comment]
Hi @rwjblue the package I updated to 0.5.2 (latest) is this https://github.com/rwjblue/ember-cli-template-lint/issues which is using the lasted version here from here `"ember-template-lint": "^0.6.0"`I posted the error here because the error is in this package. I can try to add a failing test later today if it helps.

This might be related: https://github.com/rwjblue/ember-cli-template-lint/issues/195

Thanks in advance

**Comment by rwjblue** [comment]
Oh no! I got confused and didn't notice the package name, sorry about that! I'll look into this ASAP.

**Comment by duggiefresh** [comment]
I was saw this issue, and added tests. https://github.com/rwjblue/ember-template-lint/pull/173

Strange though, they seem to pass locally for both "good" and "bad" examples. :man_shrugging:

**Comment by kellyselden** [comment]
This is resolved in v0.6.4

**Comment by rwjblue** [comment]
Thanks @kellyselden!

**Comment by kboucher** [comment]
I came across this issue in `v4.10.1` and `v5.7.2`. It seemed to go away when I removed the `indentation` specification of the `attribute-indentation` rule.

```
'attribute-indentation': {
    'indentation': 4,
},
```

...however, that turned out to be a red herring. Ultimately the issue seemed to be caused by something in the template I was trying to lint. Unfortunately, I don't know what but thought mentioning that the issue could be caused by the template might be helpful to someone else.
